#! /bin/sh /usr/share/dpatch/dpatch-run
## default_log_settings.dpatch by  <till.kamppeter@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Deactivate CUPS' own log rotating as the system already provides a
## DP: more sophisticated log rotating

@DPATCH@
diff -urNad cups-1.4.0~svn8773~/conf/cupsd.conf.in cups-1.4.0~svn8773/conf/cupsd.conf.in
--- cups-1.4.0~svn8773~/conf/cupsd.conf.in	2009-08-23 12:15:52.000000000 +0200
+++ cups-1.4.0~svn8773/conf/cupsd.conf.in	2009-08-23 18:47:44.000000000 +0200
@@ -10,6 +10,10 @@
 # for troubleshooting...
 LogLevel @CUPS_LOG_LEVEL@
 
+# Deactivate CUPS' internal logrotating, as we provide a better one, especially
+# LogLevel debug2 gets usable now
+MaxLogSize 0
+
 # Administrator user group...
 SystemGroup @CUPS_SYSTEM_GROUPS@
 @CUPS_SYSTEM_AUTHKEY@
